lib/Dancer2/Core/App.pm

```diff
@@ -1619,6 +1620,33 @@ sub response_not_found {
     return $response;
 }
 
+sub uri_for_route {
+    my ( $self, $route_name, $named_params, $unnamed_params, $dont_escape ) = @_;
```

Comment:
It looks like `$unnamed_params` and `$dont_escape` are not yet implemented.  To avoid footguns, should we croak if either is defined?  Or, better yet, just remove them from the arg list for now?  

Separately, is `$unnamed_params` actually intended to be the future query-parameters argument, similar to [`uri_for`](https://metacpan.org/pod/distribution/Dancer2/lib/Dancer2/Manual.pod#uri_for)?

Reply:
Took me time digging from the commit message and my memory to remember the issue.

I separated named parameters from unnamed parameters because if `splat` or megasplat are used, they need expanding unnamed parameters.

I think it's possible to support unnamed parameters (I just didn't because I wanted to throw a quick PR out, little did I know...) so I'll try to add that.

The escaping isn't supported but *should* be supported.

Reply:
Resolved (but GH doesn't give me a "Resolve conversation" button)